src/vpk/Velopack.Packaging/Abstractions/IPackOptions.cs

```diff
@@ -9,4 +9,5 @@ public interface IPackOptions : INugetPackCommand, IPlatformOptions
     string Exclude { get; set; }
     bool NoPortable { get; set; }
     bool NoInst { get; set; }
+    bool BuildMsi { get; }
```

Comment:
Remove property from this interface - this should primarily only be for properties common between all package builders

Reply:
Done